pipeline/init_pipeline.py

```diff
@@ -60,4 +60,4 @@ def get_clusters(deployment):
 db_utils.merge_conn(
         models.Connection(
             conn_id='Slack', conn_type='http',
-            host='localhost'))
+            host=os.environ["SLACK_NOTIFICATION_CHANNEL"]))
```

Comment:
Is there a reason to keep this field name as "host"?

Reply:
Right, I am abusing a http connection to store slack information. It is possible to use a variable instead, a connection just provide some predefined structure so we have easy access the members, the main reason I choose to use it is the username and password field. 
I think a http connection has to have a host entry, I used to set it to localhost as a placeholder, here I recycle it for the notification channel. Do you think it is better to add a field in the extra part and keep the localhost entry?
The non-hacky way to fix it is to implement our own connction type or switch to variables. The situation is similar to the `InstanceGroups` connection, a http connection is used to store the cluster information.

Reply:
Ok. I figured that this was similar to `InstanceGroups`, but I didn't know all of the mechanics, thanks. I'll look at the new commits now.

Reply:
Here's my current preference ranking:
(1) Add the channel field to the extra part of the connection - relatively easy and quite clear
(2) Switch to variables - also seems somewhat easy, but more work
(3) Subclass `Connection` for the usage type you want - clear, but requires a new concept for something pretty simple
(4) Add a couple of comment lines describing the term swap - least clear option, but clear enough

Any of these are sufficient in my mind.

Reply:
Thanks, I updated the patch to store the notification channel in the extra field. How does it look now?